### ISSUE TITLE
Updating Android x86-64 System images

### ISSUE THREAD

**Issue by juadde** [muflone/pkgbuilds#12]
Are you OK with PR like #10 ?
Can I help you maintaining these images?
If you want you can add me (Kppqju77) as co-maintainer for those images on AUR also.

**Comment by juadde** [comment]
Do you want to drop android-google-apis{,-playstore}-x86-64-system-image so I can maintain those? (looks like you got your hands full with all the other packages...)

**Comment by muflone** [comment]
Hi @juadde sorry for the late reply but I'm very busy lately and my job moved me far from my place.

I'll integrate your PR and I'll accept your kind offer to co-maintain the package. I've so many activities, and I prefer to give priority to update official packages in Arch Linux, second to the AUR.

**Comment by muflone** [comment]
Packages updated:
- android-google-apis-x86-system-image 29_r10-1
- android-google-apis-x86-64-system-image 29_r10-1
- android-google-apis-playstore-x86-system-image 29_r08-1
- android-google-apis-playstore-x86-64-system-image 29_r08-1

I wasn't able to find any updated version for android-system-image-x86 and android-system-image-x86_64

could you please share same link where to find the updated images for the standard android-x86-system-image and android-x86-64-system-image?

I'll keep this issue opened until the images are updated to version 30

**Comment by juadde** [comment]
I think you are up-to-date with those ones ;) (At least that's what android-studio says).
I will double-check, but I think you're good now.

**Comment by juadde** [comment]
Checksums on google website are sha1 only (https://dl-ssl.google.com/android/repository/sys-img/google_apis/sys-img2-1.xml) should we switch to this so everybody can check google's checksum?

I do have a working 30_r05 android-google-apis-x86-64-system-image, is it OK if I push it directly?

**Comment by muflone** [comment]
Sure go ahead updating the package, but please always build the packages using extra-x86_64-build from devtools package.

About the pending request don't worry, I'll deny it in the weekend when I'll be in front of my PC.

**Comment by juadde** [comment]
I had to run something like:
`extra-x86_64-build -- -I /var/cache/pacman/pkg/jdk-openjdk-14.0.1.u7-1-x86_64.pkg.tar.zst -I ~/.cache/yay/android-sdk/android-sdk-26.1.1-1-x86_64.pkg.tar.xz  -I ~/.cache/yay/android-sdk-platform-tools/android-sdk-platform-tools-30.0.2-1-x86_64.pkg.tar.xz  -I ~/.cache/yay/android-platform/android-platform-29_r04-1-any.pkg.tar.xz`
for extra-x86_64-build, but everything looks fine.

I also added -29 package variants if people do not want the latest version.

If you want the help for the playstore images, just add me as co-maintainer (I forgot to put my name on the start of the PKGBUILD, I will do it on the next commit).

After checking again, android-system-image-x86 and android-system-image-x86_64 are not available in sdkmanager, so for me the flag was wrong on those ones.
